src/app/public/modules/vertical-tabset/vertical-tabset-group.component.html

```diff
@@ -11,21 +12,43 @@
     [tabIndex]="-1"
   >
     {{ groupHeading }}
-    <sky-chevron *skyThemeIf="'default'"
+    <sky-chevron
       class="sky-vertical-tabset-group-header-chevron"
       [direction]="open ? 'up' : 'down'"
       [disabled]="disabled"
       (directionChange)="toggleMenuOpen()"
       (keyup.enter)="$event.stopPropagation()"
     >
     </sky-chevron>
-    <ng-container *skyThemeIf="'modern'">
+  </div>
+  <div
+    class="sky-vertical-tabset-group-content"
+    *ngIf="open"
+    [@tabSlide]
+  >
+    <ng-content></ng-content>
+  </div>
+
+  <div
+    *skyThemeIf="'modern'"
+    class="sky-vertical-tabset-group-header sky-padding-even-default"
+    [ngClass]="{
+      'sky-vertical-tabset-group-header-sub-open sky-font-emphasized': subMenuOpen(),
+      'sky-deemphasized': disabled,
+      'sky-vertical-tabset-disabled': disabled
+    }"
+    (click)="toggleMenuOpen()"
+    (keyup.enter)="toggleMenuOpen()"
+    [tabIndex]="-1"
```

Comment:
`[tabIndex]="-1"` should be before the `(click)="toggleMenuOpen()"`

Our standard ordering is 

```
<component-name *structuralDirective=""
  nativeattributes=""
  unboundProperty=""
  [inputs]=""
  [(twoWay)]=""
  (outputs)=""
  #templateReferenceVariables
>
  Some content here.
</component-name>
```

Reply:
NOTE: I see now you just copied from above - and what was there was just already wrong :)

Reply:
Think I addressed this in recent commit. https://github.com/blackbaud/skyux-tabs/pull/178/commits/f23bd3b82e0f0bcaa9face16e77ee5d8720a483d